client/components/payment-activity/style.scss

```diff
@@ -142,10 +140,48 @@
 			@include amount-styles;
 		}
 
+		&__tooltip {
+			.wcpay-inline-notice.components-notice {
+				margin-top: 8px;
+				margin-right: 0;
+				padding: 8px;
+				color: #044b7a;
```

Comment:
Do we have a color code we can use here?

Reply:
Thanks for catching that. Found the code for here and one more place and replaced it here - https://github.com/Automattic/woocommerce-payments/pull/8659/commits/892f205d5a86dc3f134229c4abc5f28bb5f8c017